src/slider/README.md

```diff
@@ -1 +1,83 @@
-# Slider widget
+# @dojo/widgets/slider/Slider widget
+
+Dojo 2's `Slider` widget creates a range slider control with a styleable track, fill, and thumb.
+
+
+## Features
+
+- Horizontal or vertical slider with a track, fill, and single thumb
+- Output node with customizable text
+
+### Accessibility Features
+
+`Slider` uses the native `<input type="range">`as its base, which ensures built-in keyboard and screen reader accessibility. All common form field attributes (`disabled`, `invalid`, `readOnly`, `required`) may be set, as well as a visible or hidden label. The output node is associated with the input.
+
+## Example Usage
+
+```typescript
+// Basic horizontal slider
+w(Slider, {
+	label: 'Choose a number from 1 to 100',
+	min: 0,
+	max: 100,
+	step: 1,
+	value: this.state.sliderValue,
+	onInput: (event: TypedTargetEvent<HTMLInputElement>) => {
+		this.setState({ sliderValue: parseFloat(event.target.value) });
+	},
+});
+
+// Horizontal slider with custom output
+w(Slider, {
+	label: 'How much do you like tribbles?',
+	min: 0,
+	max: 100,
+	output: (value: number) => {
+		if (value < 20) { return 'I am a Klingon'; }
+		if (value < 40) { return 'Tribbles only cause trouble'; }
+		if (value < 60) { return 'They\`re kind of cute'; }
+		if (value < 80) { return 'Most of my salary goes to tribble food'; }
+		else { return 'I permanently altered the ecology of a planet for my tribbles'; }
+	},
+	step: 1,
+	value: this.state.tribbleValue,
+	onInput: (event: TypedTargetEvent<HTMLInputElement>) => {
+		this.setState({ tribbleValue: parseFloat(event.target.value) });
+	}
+});
+
+// Vertical slider with validation
+w(Slider, {
+	label: 'Choose a value below 50',
+	min: 0,
+	max: 60,
+	value: verticalValue,
+	vertical: true,
+	invalid: verticalInvalid,
+	onInput: (event: TypedTargetEvent<HTMLInputElement>) => {
+		const value = parseFloat(event.target.value);
+		this.setState({
+			verticalValue: value,
+			verticalInvalid: value > 50 ? true : false
+		});
+	}
+});
+```
+
+## Theming
+
+The following CSS classes are available on the `Slider` widget for use with custom themes:
+
+- `root`: Applied to either the wrapping `<label>`, or a `<div>` in the same position in the node hierarchy
```

Comment:
Not essential but it might be nice to organize these into two sections (always, vs. conditional)?